src/index.js

```diff
@@ -0,0 +1,81 @@
+import './style.css';
+
+const buttonStart = document.querySelector('.button-start');
+const buttonStop = document.querySelector('.button-stop');
+const stopWatchScreen = document.querySelector('.stop-watch-screen');
+if (buttonStop) {
+  buttonStop.disabled = true;
+}
+
+class Stopwatch {
+  constructor() {
+    this.startTime = 0;
+    this.elapsedTime = 0;
+    this.running = false;
+    this.timer = null;
+  }
+
+  start() {
+    if (!this.running) {
+      this.startTime = Date.now() - this.elapsedTime;
+      this.running = true;
+      this.timer = setInterval(() => this.update(), 1000);
```

Comment:
It would be a bit more elegant to make `this.update` an arrow function instead of creating an arrow function here
```
this.timer = setInterval(() => this.update(), 1000);
```